distro/run/core/src/main/java/org/camunda/bpm/run/property/CamundaBpmRunProperties.java

```diff
@@ -45,6 +45,8 @@ public class CamundaBpmRunProperties {
   @NestedConfigurationProperty
   protected CamundaBpmRunRestProperties rest = new CamundaBpmRunRestProperties();
 
+  protected CamundaBpmRunDeploymentProperties deployment = new CamundaBpmRunDeploymentProperties();
```

Comment:
🔧 I know it works without this, too but it's used for some meta data:

```suggestion
  @NestedConfigurationProperty
  protected CamundaBpmRunDeploymentProperties deployment = new CamundaBpmRunDeploymentProperties();
```

Reply:
Done [here](https://github.com/camunda/camunda-bpm-platform/pull/4607/commits/96f7506e0347330a14e7566a46de552184e629ff)